stdlib/3/os/__init__.pyi

```diff
@@ -121,6 +121,8 @@ if sys.version_info >= (3, 6):
 _PathType = Union[bytes, Text]
 
 if sys.version_info >= (3, 6):
+    _PathType = Union[bytes, Text, PathLike]
```

Comment:
I'm not importing it, I'm just redefining it for Python 3.6 only to include the PathLike imported types. 

I too wondered why _PathType was underscored. I can change but thought that should be another diff?